NuGet.config

```diff
@@ -4,5 +4,6 @@
   <packageSources>
     <clear/>
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="dotnet-windowsdesktop" value="https://dotnetfeed.blob.core.windows.net/dotnet-windowsdesktop/index.json" />
```

Comment:
Why do we need this here? This should be a feed only to find arcade, nothing else.

Reply:
I made the same change in coresdk and it was approved. The feeds are listed in multiple places and I wasn't sure where the right place was. 

I asked Matt Mitchell and he said it would be safer to add it to the NuGet.config because it's tough to tell where the feeds are being read from in every scenario.

I can certainly remove it if you don't want it here :)

Reply:
Fixed